### PR TITLE
Use Maintained Compose Files

### PR DESCRIPTION
**Pull request by Mjaethers**
The standalone build scripts now use the docker compose files that are used when manually building the Oakestra components. This means there is only one set of compose files that have to be maintained.
When building the warning `pull access denied for ...` is printed for multiple containers, however as far as I can tell everything still works.
Additionally instead of creating a directory in the home directory, it is placed in a new directory wherever the script is executed. I was confused by this when I used these scripts the first time myself.

### CODE REVIEW

File: scripts/StartOakestraRoot.sh [hunk 1]
```diff
@@ -35,15 +35,15 @@ if [ -z "$SYSTEM_MANAGER_URL" ]; then
     echo Default node IP: $SYSTEM_MANAGER_URL
 fi
 
-mkdir ~/oakestra 2> /dev/null
+mkdir root_orchestrator 2> /dev/null
 
-cd ~/oakestra 2> /dev/null
+cd root_orchestrator 2> /dev/null
```
[the remark this answers is not in the document]
Comment (giobart) [reply scripts/StartOakestraRoot.sh:40]: the problem I have with this approach is that depending on where they execute the script the compose files are spread around the filesystem, and the stop command changes depending on that. When the startup and stop will be managed (@Malyuk-A will potentially execute these scripts as part of the CLI installation) a user does not even need to know where the compose files are, so it makes sense to keep them in a fixed pre-dermined folder right? Lmk what you think

[the remark this answers is not in the document]
Comment (Mjaethers) [reply scripts/StartOakestraRoot.sh:40]: Okay, that makes sense. I'll be sure to clarify that this is placed in the home directory in the documentation then

File: scripts/StartOakestraCluster.sh [hunk 2]
```diff
@@ -112,13 +112,13 @@ if [ -z "$SYSTEM_MANAGER_URL" ]; then
...
-mkdir ~/oakestra 2> /dev/null
+rm -rf ~/oakestra/cluster_orchestrator 2> /dev/null
+mkdir -p ~/oakestra/cluster_orchestrator 2> /dev/null
 
-cd ~/oakestra 2> /dev/null
+cd ~/oakestra/cluster_orchestrator 2> /dev/null
 
 curl -sfL https://raw.githubusercontent.com/oakestra/oakestra/$OAKESTRA_BRANCH/scripts/utils/downloadConfigFiles.sh > downloadConfigFiles.sh
-curl -sfL https://raw.githubusercontent.com/oakestra/oakestra/$OAKESTRA_BRANCH/run-a-cluster/cluster-orchestrator.yml > cluster-orchestrator.yml
+curl -sfL https://raw.githubusercontent.com/oakestra/oakestra/$OAKESTRA_BRANCH/cluster_orchestrator/docker-compose.yml > cluster-orchestrator.yml
```
Comment (giobart) [inline scripts/StartOakestraCluster.sh:121]: this only works if people actually clone the Oakestra repo. 

`cluster_orchestrator/docker-compose.yml` manually builds the services.
while
`run-a-cluster/cluster-orchestrator.yml` should use the pre-built images.

With this script we don't want the user to clone Oak repo but this should run directly.

File: scripts/StartOakestraRoot.sh [hunk 3]
```diff
@@ -35,15 +35,14 @@ if [ -z "$SYSTEM_MANAGER_URL" ]; then
...
 
-mkdir ~/oakestra 2> /dev/null
-
-cd ~/oakestra 2> /dev/null
+mkdir -p ~/oakestra/root_orchestrator 2> /dev/null
+cd ~/oakestra/root_orchestrator 2> /dev/null
 
 curl -sfL https://raw.githubusercontent.com/oakestra/oakestra/$OAKESTRA_BRANCH/scripts/utils/downloadConfigFiles.sh > downloadConfigFiles.sh
-curl -sfL https://raw.githubusercontent.com/oakestra/oakestra/$OAKESTRA_BRANCH/run-a-cluster/root-orchestrator.yml > root-orchestrator.yml
+curl -sfL https://raw.githubusercontent.com/oakestra/oakestra/$OAKESTRA_BRANCH/root_orchestrator/docker-compose.yml > root-orchestrator.yml
```
Comment (giobart) [inline scripts/StartOakestraRoot.sh:42]: Same problem as the cluster orchestrator script.

**Review by giobart — CHANGES_REQUESTED** [review]

**Comment by giobart** [comment]
> The standalone build scripts now use the docker compose files that are used when manually building the Oakestra components.

Mybe there is a misunderstanding.
This should not be the case in my opinion. We don't want to build Oakestra every time we need to deploy it but rather use the pre-built images we have. The run-a-cluster scripts serve this specific purpose, they use the pre-built images.

**Comment by Mjaethers** [comment]
In terms of testing:
This is a bit of a chicken-and-egg problem, the script can't pull the compose override file as it was only created by this commit and has not been pushed to the main branch.
When testing with the new override files manually it works.

**Review by giobart — APPROVED** [review]

**Comment by giobart** [comment]
LGTM! :)

**Pull request merged**